Packs/Code42/Integrations/Code42/Code42.py

```diff
@@ -230,11 +230,11 @@ def remove_user_risk_tags(self, username, risk_tags):
         self._get_sdk().detectionlists.remove_user_risk_tags(user_id, risk_tags)
         return user_id
 
-    def get_all_high_risk_employees(self, risk_tags, results):
+    def get_all_high_risk_employees(self, risk_tags, results, filter_type):
         risk_tags = _try_convert_str_list_to_list(risk_tags)
         results = int(results) if results else None
         res = []
-        pages = self._get_sdk().detectionlists.high_risk_employee.get_all()
+        pages = self._get_sdk().detectionlists.high_risk_employee.get_all(filter_type=filter_type)
```

Comment:
This will cause `None` to be sent as the filter type if one isn't passed in, which would override the default value of `"OPEN"`. We should set `filter_type` to `"OPEN"` if it is falsy before calling this.

Reply:
I think the default value for the command is OPEN, but it is good to be double-y sure

Reply:
it is, but if you pass `None`, you will be using that and _not_ `OPEN` :-)